R/qenv-get_code.R

```diff
@@ -50,7 +50,7 @@ setMethod("get_code", signature = "qenv.error", function(object, ...) {
       sprintf(
         "%s\n\ntrace: \n %s\n",
         conditionMessage(object),
-        paste(lang2calls(object$trace), collapse = "\n ")
```

Comment:
if this is correct (which I am not quite sure yet) - please track down other usages of that function with the working hypothesis that this is the only one

Reply:
There is a lot, as we used this function to divide long expressions/languages into calls. Currently `object@trace` is equal to  `object@code` up to the last possible element of if that did throw the error. lang2calls is the successor of `format_expression` that we used to clean up expressions from `{`. I think we did to much quick substitutions of `format_expression` with `lans2calls` without further investigation and now we need to get rid of it in places where this does not belong.

Reply:
> if this is correct (which I am not quite sure yet) - please track down other usages of that function with the working hypothesis that this is the only one

@pawelru I don't quite understand what you mean. Is there a problem with `lang2calls` itself?

Reply:
I meant to check if this is the only occurrence of this function. If it happens to be true then we can remove importing / defining it. But if there are other occurrences then no action needed.

Reply:
I believe it's quite necessary when _storing_ code, which is why it can be omitted when retrieving it.

Reply:
then please ignore my comment